crawler/adapters/twitter.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'twitter'
+
+module Adapters
+  class Twitter < Base
+    def fetch(hashtag:, amount:)
+      client.search("#{hashtag}", result_type: :recent).take(amount).map do |tweet|
+        Post.new tweet.text, build_url(tweet.url), :twitter
+      end
+    end
+
+    def build_url(url)
+      "#{url.scheme}://#{url.host}#{url.path}"
+    end
+
+    def client
```

Comment:
I think this being `private` would be better

Reply:
You're right. Will change.